redaxo/src/core/fragments/core/form/searchfield.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * @var rex_fragment $this
+ * @psalm-scope-this rex_fragment
+ */
+
+$id = $clear = $class = $placeholder = '';
+$class = $this->getVar('class', 'input-group input-group-xs has-feedback form-clear-button');
```

Comment:
ist das ein tippfehler?

```suggestion
$class = $this->getVar('class', 'input-group input-group-xs has-feedback form-control-clear');
```

Reply:
hmm nee das ist wohl richtig wie es ist.. die klassennamen sind verwirrend, weil so ähnlich

Reply:
ja das eine ist der button für den clear